src/v2i-hub/SpatPlugin/test/Main.cpp

```diff
@@ -0,0 +1,14 @@
+/*
```

Comment:
Since we added unit tests to this plugin lets update the sonar.properties file to take coverage credit for the additional coverage.

Reply:
https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/334/commits/dc6c71c5e57f2e367dd1db0cdeb0460870bfb01b